#Posição
projeto_semear/Pose Posicao

---
#qual container escolhi:

uint8 container_escolhido # 0=esquerda, 1 = direita, 2 = só tem vermelho aqui

